### PR TITLE
AWS Resource Tags

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
This PR adds a section related to tags for AWS resources. 
🎫 [PCP-753](https://spectrocloud.atlassian.net/browse/PCP-753)
💻 [Preview](https://deploy-preview-1105--docs-spectrocloud.netlify.app/clusters/public-cloud/aws/architecture#tags)

CleanShot 2023-02-03 at 10 17 23

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1105#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by sadysnaat — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1105#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1105#pullrequestreview-1284012804" title="Approved">sadysnaat</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by ritawatson — APPROVED** [review]
There's no period after the two examples in the table. Should we say this style is fine when it's not a complete sentence and the example is in the code format such as in the table?

Example: `mycluste2r-vpc`

LGTM

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1105#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1105#pullrequestreview-1285659325" title="Approved">ritawatson</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1105#pullrequestreview-1284012804" title="Approved">sadysnaat</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04-clusters/01-public-cloud/01-aws/02-architecture.md
Comment (ritawatson) [inline content/docs/04-clusters/01-public-cloud/01-aws/02-architecture.md:95]:
```suggestion
In addition to the custom tags provided by you, Palette-provisioned AWS resources will receive the following default tags.
```

**Review by ritawatson — APPROVED** [review]
LGTM

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1105#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1105#pullrequestreview-1285684197" title="Approved">ritawatson</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1105#pullrequestreview-1284012804" title="Approved">sadysnaat</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04-clusters/01-public-cloud/01-aws/02-architecture.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/01-aws/02-architecture.md:100]:
```suggestion
| `kubernetes.io/cluster/[clusterName]`.                       | owned         | This tag only applies to cluster nodes. Used for Palette internal purposes to help manage the lifecycle of the cluster. |
```

File: content/docs/04-clusters/01-public-cloud/01-aws/02-architecture.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/01-aws/02-architecture.md:101]:
```suggestion
| `sigs.k8s.io/cluster-api-provider-aws/cluster/[clusterName]` | owned         | Used for Palette internal purposes to help manage the lifecycle of the cluster. |
```

File: content/docs/04-clusters/01-public-cloud/01-aws/02-architecture.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/01-aws/02-architecture.md:102]:
```suggestion
| `sigs.k8s.io/cluster-api-provider-aws/role`                  | common        | Used for Palette internal purposes to help manage the lifecycle of the cluster. |
```

**Pull request merged**
